dumpling/export/config.go

```diff
@@ -528,6 +534,28 @@ func ParseTableFilter(tablesList, filters []string) (filter.Filter, error) {
 	return filter.NewTablesFilter(tableNames...), nil
 }
 
+func GetConfTables(conf *Config, tablesList []string) (DatabaseTables, error) {
+	dbTables := DatabaseTables{}
+	var (
+		schema, name string
+		tableType    TableType
+		avgRowLength uint64
+	)
+	tableType = TableTypeBase
+	avgRowLength = 0
+	conf.specifiedTables = len(tablesList) > 0
```

Comment:
i thinks this line should put out of this func becasue its a set operate not get